### DISCUSSION TITLE
Overload driver code with out-of-tree one

### DISCUSSION THREAD

**Discussion by RomainPelletant** [zephyrproject-rtos/zephyr#54863] [Q&A] [answered]
Hello,

In a zephyr project, we have drivers with the same name than existing ones in zephyr/driver repository : do you know if it is possible to overload existing drivers by out-of-tree drivers?

Thank you

**Comment by henrikbrixandersen** [comment, marked as answer]
My suggestion would be to use different Kconfig and/or DTS compatible strings for your out-of-tree drivers.

**Reply by RomainPelletant** [reply]
Will try that. Thanks @henrikbrixandersen for your help
